content/blog/2019-02-23-is-react-translated-yet.md

```diff
@@ -1,76 +1,76 @@
 ---
-title: "Is React Translated Yet? ¡Sí! Sim! はい！"
+title: "¿Ya está traducido React? ¡Sí! Sim! はい！"
 author: [tesseralis]
 ---
 
-We’re excited to announce an ongoing effort to maintain official translations of the React documentation website into different languages. Thanks to the dedicated efforts of React community members from around the world, React is now being translated into *over 30* languages! You can find them on the new [Languages](/languages) page.
+Nos emociona poder anunciar un esfuerzo continuo para mantener traducciones oficiales en diferentes idiomas del sitio web de la documentación de React. Gracias a los esfuerzos de los miembros de la comunidad de React de todo el mundo, ¡React está siendo traducido a *cerca de 30* idiomas! Puedes encontrarlos en la nueva página de [Idiomas](/languages).
 
-In addition, the following three languages have completed translating most of the React Docs! 🎉
+¡Aún más, los tres idiomas siguientes han completado la traducción de la mayoría de la documentación de React! 🎉
 
-* **Spanish: [es.reactjs.org](https://es.reactjs.org)**
-* **Japanese: [ja.reactjs.org](https://ja.reactjs.org)**
-* **Brazilian Portuguese: [pt-br.reactjs.org](https://pt-br.reactjs.org)**
+* **Español: [es.reactjs.org](https://es.reactjs.org)**
+* **Japonés: [ja.reactjs.org](https://ja.reactjs.org)**
+* **Portugués de Brasil: [pt-br.reactjs.org](https://pt-br.reactjs.org)**
 
-Special congratulations to [Alejandro Ñáñez Ortiz](https://github.com/alejandronanez), [Rainer Martínez Fraga](https://github.com/carburo), [David Morales](https://github.com/dmorales), [Miguel Alejandro Bolivar Portilla](https://github.com/Darking360), and all the contributors to the Spanish translation for being the first to *completely* translate the core pages of the docs!
+¡Felicitaciones especiales a [Alejandro Ñáñez Ortiz](https://github.com/alejandronanez), [Rainer Martínez Fraga](https://github.com/carburo), [David Morales](https://github.com/dmorales), [Miguel Alejandro Bolivar Portilla](https://github.com/Darking360) y a todos los que contribuyen a la traducción al español por ser los primeros en traducir *completamente* las páginas principales de la documentación!
 
-## Why Localization Matters {#why-localization-matters}
+## Por qué importa la localización {#why-localization-matters}
 
-React already has many meetups and conferences around the world, but many programmers don't use English as their primary language. We’d love to support local communities who use React by making our documentation available in most popular languages.
+React tiene ya muchos encuentros y conferencias alrededor del mundo, pero muchos programadores no usan el inglés como su idioma principal. Nos encantaría apoyar a las comunidades locales que utilizan React al hacer disponible nuestra documentación en los idiomas más populares.
```

Comment:
I'd think that we could use the word `meetups` instead of `encuentros` - since we're talking about the events that are organized via meetup.com (at least, that's what I think). In Colombia we do not translate `meetups`.

Thoughts on this?

```suggestion
React tiene ya muchos meetups y conferencias alrededor del mundo, pero muchos programadores no usan el inglés como su idioma principal. Nos encantaría apoyar a las comunidades locales que utilizan React al hacer disponible nuestra documentación en los idiomas más populares.
```